cognite_toolkit/_builtin_modules/examples/cdf_location_example/locations/example.LocationFilter.yaml

```diff
@@ -0,0 +1,54 @@
+externalId: '{{ location_external_id }}'
+name: '{{ location_name }}'
+parentId: '{{ location_parent_id }}'
+description: '{{ location_description }}'
+dataModels:
+ - externalId: '{{ location_data_model_external_id }}'
+   space: '{{ location_data_model_space }}'
+   version: '{{ location_data_model_version }}'
+instanceSpaces:
+ - '{{ location_instance-space}}'
+scene:
+ externalId: '{{  location_scene_external_id }}'
+ space: '{{  location_scene_space }}'
+views:
+ externalId: '{{  location_view_external_id }}'
+ space: '{{ location_view_space }}'
+ version: '{{ location_view_version }}'
+ representsEntity: '{{ location_view_represents_entity }}'
+# assetCentric:
+#  assets:
+#    dataSetExternalIds:
+#      - '{{  location_asset_dataset_external_id }}'
+#    assetSubtreeIds:
+#      - externalId: '{{  location_asset_subtree_external_id }}'
+#    externalIdPrefix: asset-prefix
+#  events:
+#    dataSetExternalIds:
+#      - '{{  location_event_dataset_external_id }}'
+#    assetSubtreeIds:
+#      - externalId: '{{  location_asset_subtree_external_id }}'
+#    externalIdPrefix: event-prefix
+#  files:
+#    dataSetExternalIds:
+#      - '{{  location_file_dataset_external_id }}'
+#    assetSubtreeIds:
+#      - externalId: '{{  location_asset_subtree_external_id }}'
+#    externalIdPrefix: file-prefix
+#  timeseries:
+#    dataSetExternalIds:
+#      - '{{  location_timeseries_dataset_external_id }}'
+#    assetSubtreeIds:
+#      - externalId: '{{  location_asset_subtree_external_id }}'
+#    externalIdPrefix: '{{  location_timeseries_prefix }}'
+#  sequences:
+#    dataSetExternalIds:
+#      - '{{  location_timeseries_dataset_external_id }}'
+#    assetSubtreeIds:
+#      - externalId: '{{  location_asset_subtree_external_id }}'
+#    externalIdPrefix: '{{  location_sequences_prefix }}'
+#  dataSetExternalIds:
+#    - '{{  location_dataset_external_id }}'
+#  assetSubtreeIds:
+#    - externalId: '{{  location_asset_subtree_external_id }}'
+#  externalIdPrefix: '{{  location_general_external_id_prefix }}'
```

Comment:
I would further split this into two, I guess asset centric with individual filters should not be combined with shared filter.